x-pack/elastic-agent/pkg/agent/cmd/container.go

```diff
@@ -452,17 +453,24 @@ func kibanaFetchToken(cfg setupConfig, client *kibana.Client, policy *kibanaPoli
 func kibanaClient(cfg kibanaConfig, headers map[string]string) (*kibana.Client, error) {
 	var tls *tlscommon.Config
 	if cfg.Fleet.CA != "" {
+		var err error
 		tls = &tlscommon.Config{
 			CAs: []string{cfg.Fleet.CA},
 		}
+		if err != nil {
+			return nil, err
+		}
```

Comment:
Why was this added? Don't see how `var err error` is ever set here.

Reply:
looks like a merge gone wrong.